### DISCUSSION TITLE
RE: Ideas on Soldering and Lots of Play

### DISCUSSION THREAD

**Discussion by silver2row** [phil-barrett/grblHAL-teensy-4.x#63] [General]
Hello,

This may seem like an unimportant question but I think it is worthy of some attention.

1. There is a lot of play in between the pins and OD of the holes to be soldered.
2. What size solder are you using?
3. Is there a way to connect the terminals in a way, esp. those getting more movement, that prevents their loosening?

Anyway, that is all for now. I am a novice w/ this piece of electronics so far. I cannot wait to solder everything together and get crackin' on making machines available to me.

Seth

P.S. When you have time, please take time out to answer.

**Comment by silver2row** [comment]
Hello Again,

I have a 12v PWM enabled laser that I would like to test fire for starters. The PWM enable pin is for 5v logic while the powering of the device is 12v. Can this board work w/ my current configuration of a laser?

Seth

**Reply by silver2row** [reply]
Hello,

Okay...I see ` now ` how the terminals on the kit fit together. I am not so worried now about how to solder or what type of solder to use. This is done now. I am just still wondering about the second comment about the 12v laser w/ 5v PWM logic.

Seth

**Comment by phil-barrett** [comment]
Glad you figured out the screw terminals.  I made the holes slightly oversized to insure against variations in manufacturing of the terminals.

As to directly driving a 12V input on the laser - sorry but that probably will not work. Does your module have PWM input? That might be 5V compatible.

**Reply by silver2row** [reply]
Hello,

The PWM logic is 5v but to power the device, I need 12v power and then a GND.

Seth

**Comment by phil-barrett** [comment]
Any 12V power will suffice.  You can share it with the T41U5XBB if you need it for relays.  All the Gnds on the board are connected.

**Reply by silver2row** [reply]
Hello Sir,

Thank you for the info. about the board. I will keep reading the manual you put together. Thank you again.

Seth
